independent-publisher-connectors/Shopify Admin Content/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2895 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Shopify Admin Content",
+    "description": "The Shopify REST Admin API lets you build apps and other integrations for the Shopify admin.",
+    "contact": {
+      "name": "Hitachi Solutions",
+      "email":"jlindstrom@hitachisolutions.com"
+    }
+  },
+  "host": "thatapiguy.myshopify.com",
```

Comment:
@thatapiguy Is it a valid host URL ?

Reply:
I will change this to yourstorename.myshopify.com as the user will be prompted to enter the host URL as a connection parameter.